morepath/core.py

```diff
@@ -20,24 +20,21 @@
 your code with future version of Morepath if you do that, though.
 """
 
-import dectate
-import importscan
+import dectate  # noqa
```

Comment:
It looks like it should be possible to remove ``import dictate``. Can you try again?